src/main/java/org/javarosa/core/model/instance/geojson/GeojsonGeometry.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 ODK
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.javarosa.core.model.instance.geojson;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import java.util.ArrayList;
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class GeojsonGeometry {
+    private String type;
+    private ArrayList<String> coordinates;
+
+    public String getType() {
+        return type;
+    }
+
+    public String getOdkCoordinates() {
+        if (!(type.equals("Point") && coordinates.size() == 2)) {
+            throw new UnsupportedOperationException("Only Points are currently supported");
```

Comment:
Not going to be a problem for long, but maybe we should just skip unsupported elements rather than exploding? That would let us test in Collect with `.geojson` files without worrying about it crashing the app.

Reply:
I think we likely will have a release with only points. In that case it feels better to fail for reasons similar to the "fast external itemsets" bug you identified -- it's not great if a subset of user features are silently filtered out. I don't feel strongly about it but that was my reasoning for exploding.

Reply:
That makes sense. Let's use a checked exception then, so there's not a risk of Collect missing this and it resulting in a crash.